Packages/com.llealloo.audiolink/Runtime/Scripts/ThemeColorController.cs

```diff
@@ -170,17 +216,15 @@ public void InitializeAudioLinkThemeColors()
         {
             if (audioLink == null) return;
 
+            Color[] customThemeColors = GetCustomThemeColors();
+
             customThemeColors[0] = audioLink.customThemeColor0;
             customThemeColors[1] = audioLink.customThemeColor1;
             customThemeColors[2] = audioLink.customThemeColor2;
             customThemeColors[3] = audioLink.customThemeColor3;
 
-            _initCustomThemeColors = new[] {
-                customThemeColors[0],
-                customThemeColors[1],
-                customThemeColors[2],
-                customThemeColors[3],
-            };
+            //shallow copy of the array
+            _initCustomThemeColors = GetCustomThemeColors();
```

Comment:
this isn't a shallow copy since Color is a value type?

Reply:
right but the array isnt a reference to the real array here, its a copy of the array itself. I could just remove the word shallow

Reply:
sounds good to me